Help/MAC_Hints.rst

```diff
@@ -1,4 +1,4 @@
-Coverage Installation Hints for OSX Users:
```

Comment:
the commit message needs some work: First of all, imperative mood in the shortlog
Changed -> Change